benchmarks/dbscan/dbscan.cpp

```diff
@@ -68,6 +68,7 @@ int main(int argc, char *argv[])
       ( "help", "help message" )
       ( "algorithm", bpo::value<std::string>(&params.algorithm)->default_value("dbscan"), "algorithm (dbscan | mst)" )
       ( "filename", bpo::value<std::string>(&params.filename), "filename containing data" )
+      ( "backend", bpo::value<std::string>(&params.backend)->default_value("default"), "Kokkos backend (serial | openmp | cuda | hip | sycl | threads | openmptarget)" )
```

Comment:
I would add the default value `default` to the list of backends to choose from.
```suggestion
      ( "backend", bpo::value<std::string>(&params.backend)->default_value("default"), "Kokkos backend (default | serial | openmp | cuda | hip | sycl | threads | openmptarget)" )
```
Also, generating the list based on the enabled backends would be nice.